### PR TITLE
mkcloud: Adds steps to run mkcloud on mkcloud.cloud.suse.de

### PR DESCRIPTION
**Pull request by sayalilunkad**

### CODE REVIEW

File: docs/mkcloud.md [hunk 1]
```diff
@@ -147,6 +147,22 @@ compute, 1 for controller).
 If you want to run `tempest` in an already created cloud you can run the
 following command:
 
+
+Running mkcloud on mkcloud.cloud.suse.de host
+
+* ssh to mkcloud.cloud.suse.de, user root with default password.
```
[the remark this answers is not in the document]
Comment (aspiers) [reply docs/mkcloud.md]: There is already a document somewhere in GitHub which covers the allocations file, so please don't duplicate to that - just link to it.  I can't remember where it is but I'm 95% sure I linked to it in a recent post to cloud-devel or cloud-team.

[the remark this answers is not in the document]
Comment (aspiers) [reply docs/mkcloud.md]: Also please bear in mind that some of our partners (even customers?) are using mkcloud, so mixing up pure mkcloud docs with internal stuff specific to SUSE is not necessarily a good idea (although this repo is already a mess in that respect).

[the remark this answers is not in the document]
Comment (sayalilunkad) [reply docs/mkcloud.md]: I have tried looking for the document but don't find it in the mails or anywhere. If you can find it please link it here. Also I agree this is probably not the best place to have internal stuff, would doc-cloud be a better place for this?

[the remark this answers is not in the document]
Comment (aspiers) [reply docs/mkcloud.md]: Here it is:

http://mailman.suse.de/mlarch/SuSE/cloud-devel/2018/cloud-devel.2018.07/msg00194.html

Please read https://jira.suse.de/browse/SCRD-2704.  Currently https://github.com/SUSE/cloud/tree/master/docs is the default location for internal developer docs.

**Comment by sayalilunkad** [comment]
@aspiers thanks! I'll close this in favor of this doc https://github.com/SUSE/cloud/blob/master/docs/dev_env/setup_cloud_on_mkcloud_host.md which covers everything needed!

**Comment by sayalilunkad** [comment]
@aspiers yes, agreed.

**Pull request closed without merging**